lib/matplotlib/axes/_axes.py

```diff
@@ -5120,15 +5120,24 @@ def imshow(self, X, cmap=None, norm=None, aspect=None,
 
         """
 
+        temp = np.asarray(X)
+        if temp.ndim == 3 and isinstance(norm, mcolors.BivariateNorm):
+                    temp = norm(temp)
+                    X = cmap(temp, alpha=self.get_alpha(), bytes=True)
```

Comment:
Doing this here means you can not change anything about the color map or norm later.